webpack.standalone.config.js

```diff
@@ -18,22 +22,34 @@ module.exports = {
 		],
 	},
 	plugins: [
-		new LodashModuleReplacementPlugin( {
-			paths: true,
-			shorthands: true,
-		} ),
 		new webpack.DefinePlugin( {
-			'process.env.NODE_ENV': JSON.stringify( 'development' ),
+			'process.env.NODE_ENV': JSON.stringify( env ),
 		} ),
 	],
 	resolve: {
 		extensions: [ '.js', '.jsx' ],
 		modules: [ path.resolve( __dirname ), path.resolve( __dirname, 'node_modules' ) ],
 	},
-	devServer: {
-		contentBase: './targets/standalone/public',
-		compress: true,
-		port: 9000,
-		hot: true,
-	},
 };
+
+switch ( env ) {
+	case 'development':
+		config.devtool = 'source-map';
+		config.devServer = {
+			contentBase: path.resolve( __dirname, 'targets/standalone' ),
+			publicPath: '/',
+			port: 9000,
+		};
+		break;
+
+	case 'production':
+		config.plugins.push( new webpack.optimize.ModuleConcatenationPlugin() );
+		config.plugins.push( new UglifyJsPlugin() );
+		config.plugins.push( new LodashModuleReplacementPlugin( {
```

Comment:
We no longer use `lodash-webpack-plugin`: it introduced subtle bugs and the size reduction it provided wasn't worth the hassle of changing our use of lodash to make it safe.

Reply:
Will remove it.